### ISSUE TITLE
Скрипт не запускается автоматом

### ISSUE THREAD

**Issue by alena-dem** [SBProX/SBProX-Dev#1102]
Здравствуйте, скрипт не запускается автоматически
![2020-04-14_09-20-46](https://user-images.githubusercontent.com/63700789/79304509-37202a00-7efa-11ea-9fff-47bc5b055ca8.png)

**Comment by OlegT911** [comment]
Так же происходит при первом включении. Если перезапустить квик второй раз, скрипт запуститься автоматически.
![2020-04-13 09 36 22](https://user-images.githubusercontent.com/60439200/79319186-a99c0480-7f10-11ea-8246-009f88951496.jpg)

**Comment by alena-dem** [comment]
А с первого раза нельзя сделать?



Отправлено из мобильной Почты Mail.ru


среда, 15 апреля 2020 г., 12:00 +0300 от notifications@github.com  <notifications@github.com>:

**Comment by OlegT911** [comment]
Скрипт должен запускаться автоматически, вместе с квиком. Здесь явно присутствует баг.

**Comment by alena-dem** [comment]
Это можно исправить?



Отправлено из мобильной Почты Mail.ru


среда, 15 апреля 2020 г., 23:17 +0300 от notifications@github.com  <notifications@github.com>:

**Comment by test-777** [comment]
Регулярно так как и у автора скрипт при включении квика находиться в состоянии (выкл).
Где то в 30% запусков его приходиться активизировать в ручную. При этом скрипт кускальпа такой динамики не показывает  и повторного включения не требует.

Конечно можно включать и ручками - но не удобно.

**Comment by OlegT911** [comment]
Это не только можно, но и нужно исправить. Надеюсь разработчики обратят внимание.

**Comment by SBProX** [comment]
Сначала необходимо запускать скрипт, а потом SBPro - тогда такой проблемы не будет.

**Comment by OlegT911** [comment]
Сегодня квик потерял связь на несколько секунд, после восстановления скрипт не запустился. Потом пришлось перегрузить СБпро. Вам не кажется что так не должно быть?

**Comment by SBProX** [comment]
> Сегодня квик потерял связь на несколько секунд, после восстановления скрипт не запустился. Потом пришлось перегрузить СБпро. Вам не кажется, что так не должно быть?

Безусловно

**Comment by alena-dem** [comment]
Даже при очередности запуска квик, потом сб про, скрипт не всегда запускается, запускаю вручную

**Comment by test-777** [comment]
Подтверждаю - у меня так же при соблюдении очередности запуска, скрипт все равно приходиться запускать в ручную.

**Comment by SBProX** [comment]
Обновите скрипт, перезалили по ссылке: https://drive.google.com/open?id=1GnYZNI79JlJKjmQWNW5wiqBz5Dq2khZx

Теперь должно все работать.

**Comment by alena-dem** [comment]
> Обновите скрипт, перезалили по ссылке: https://drive.google.com/open?id=1GnYZNI79JlJKjmQWNW5wiqBz5Dq2khZx
> 
> Теперь должно все работать.
Тоже самое, второй день.

**Comment by SBProX** [comment]
Скрипт по прежнему не перезапускается с той же ошибкой?
Вы точно скачали скрипт по ссылке?

**Comment by alena-dem** [comment]
> Скрипт по прежнему не перезапускается с той же ошибкой?
> Вы точно скачали скрипт по ссылке?

Да,  скачала , про ошибку не уверена, но не каждый день ломается

**Comment by alena-dem** [comment]
![2020-06-01_09-26-25](https://user-images.githubusercontent.com/63700789/83381721-0fbde780-a3ea-11ea-99ef-4e8fe537705c.png)
Вот сегодня
